### PR TITLE
TP-6556: add contact_number to CRM

### PR DESCRIPTION
**Pull request by Tr4pSt3R**
depends on https://github.com/moneyadviceservice/cream/pull/13

### CODE REVIEW

**Comment by tomas-stefano** [comment]
:+1:

File: lib/core/entity/customer.rb [hunk 1]
```diff
@@ -33,7 +34,8 @@ def to_crm_hash
         GenderCode: { Value: GENDER_MAP[gender] },
         mas_AgeRange: { Value: AGE_RANGES_MAP[age_range] },
         BirthDate: iso8601_date_of_birth,
-        DoNotBulkEMail: !newsletter_subscription
+        DoNotBulkEMail: !newsletter_subscription,
+        Telephone2: contact_number
```
Comment (asmega) [inline lib/core/entity/customer.rb]: was `Telephone` used for something else?

Comment (Tr4pSt3R) [inline lib/core/entity/customer.rb]: Telephone2 is how it is named in CRM. That's the logical name that the CRM guys sent across. 
screen shot 2015-08-03 at 09 54 55

**Comment by asmega** [comment]
:+1:

**Comment by neoeno** [comment]
:+1:

**Pull request merged**